client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/email-validation-banner.tsx

```diff
@@ -0,0 +1,73 @@
+import { Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useResendEmailVerification } from 'calypso/landing/stepper/hooks/use-resend-email-verification';
+
+interface EmailValidationBannerProps {
+	email: string;
+	closeBanner: () => void;
+}
+
+const CheckmarkIcon = () => (
```

Comment:
In case we're not able to locate an existing Icon component for this, would it be valuable to move this out to its own component, and additionally we can reuse where needed?